vtm-android-example/build.gradle

```diff
@@ -1,6 +1,7 @@
 apply plugin: 'com.android.application'
 
 dependencies {
+    compile project(':vtm')
```

Comment:
We shouldn't need that as have already the `vtm-android` being a transitive dependency.